flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/utils/ChunkUtils.java

```diff
@@ -42,20 +42,20 @@ public static RowType getChunkKeyColumnType(Table table, @Nullable String chunkK
         return getChunkKeyColumnType(getChunkKeyColumn(table, chunkKeyColumn));
     }
 
-    public static RowType getChunkKeyColumnType(Column chunkKeyColumn) {
+    public static RowType getChunkKeyColumnType(@Nullable Column chunkKeyColumn) {
+        if (chunkKeyColumn == null) {
+            return (RowType) ROW().getLogicalType();
+        }
```

Comment:
This should not be changed as this. I think we could use the first column as the chunk key if the chunk key is not set in the table options.

Reply:
You mean using the first column as chunk key to split the table if it has no primary key ?

Reply:
I replied about this part at the first comment.